solcjs

```diff
@@ -45,6 +48,26 @@ function abort (msg) {
   process.exit(1);
 }
 
+if (argv['standard-json']) {
+  if (!solc.supportsStandard) {
+    abort('Compiler does not support Standard JSON I/O');
+  }
+
+  var size = fs.fstatSync(process.stdin.fd).size;
```

Comment:
Don't know, took this code from stackoverflow and it works (it counts till EOF).